redis/redis.go

```diff
@@ -115,3 +115,24 @@ func ReceiveWithTimeout(c Conn, timeout time.Duration) (interface{}, error) {
 	}
 	return cwt.ReceiveWithTimeout(timeout)
 }
+
+// SlowLog represents a redis SlowLog
+type SlowLog struct {
+	// ID is a unique progressive identifier for every slow log entry.
+	ID int64
+
+	// UnixTime is the unix timestamp at which the logged command was processed.
```

Comment:
A time.Time would be more idiomatic

Reply:
> A time.Time would be more idiomatic

Since this is a epochtime (number of seconds that have elapsed since January 1, 1970) its better we define it as int64 and callers can convert it into the time format they want.

Reply:
I'm afraid I disagree, in the same vain as you convert the execution time into a `time.Duration`, its just much more useful to have it in the idiomatic type where the user can then easily call `time.Unix()` if they want the the epoch, avoiding most consumers having to do the conversion themselves.

Reply:
Makes sense. changed to time.Time